Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -409,8 +458,12 @@ private ResultsViewModel SelectedResults
         public ICommand ReloadPluginDataCommand { get; set; }
         public ICommand ClearQueryCommand { get; private set; }
         public ICommand AutocompleteQueryCommand { get; set; }
+        public ICommand ReverseHistory { get; set; }
+        public ICommand ForwardHistory { get; set; }
+        public ICommand InsertLastQuery { get; set; }
```

Comment:
these command variable naming should follow the others- e.g. ReverseHistoryCommand